packages/react-native/React/UIUtils/RCTUIUtils.m

```diff
@@ -24,20 +24,12 @@ RCTDimensions RCTGetDimensions(RCTPlatformView *rootView)
   // We fallback to screen size if a key window is not found.
   CGSize windowSize = mainWindow ? mainWindow.bounds.size : screenSize;
 #else // [macOS
-  NSWindow *window = nil;
-  NSSize windowSize;
-  NSSize screenSize;
-  if (rootView != nil) {
-	  window = [rootView window];
-	  windowSize = [window frame].size;
-  } else {
-    // We don't have a root view so fall back to the app's key window
-    window = [NSApp keyWindow];
-    windowSize = [window frame].size;
-  }
-  screenSize = [[window screen] frame].size;
+  NSWindow *window = [NSApp keyWindow];
+  NSSize windowSize = window ? [window frame].size : CGSizeMake(0, 0);
+  NSSize screenSize = window ? [[window screen] frame].size : CGSizeMake(0, 0);
+  CGFloat scale = window ? [[window screen] backingScaleFactor] : 1.0; // Default scale to 1.0 if window is nil
+  CGFloat fontScale = 1;
```

Comment:
I'm assuming this will need to change in the future for Fabric?

Reply:
Why?

Reply:
I moved this to the shared implementation w/ iOS but I'm still defaulting to 1.0
I was assuming we use the same accessibility api's but `accessibilityManager.multiplier` always returned zero on MacOS.

https://github.com/facebook/react-native/blob/40c149b0c4354564c305378aebdc79b9b4d34e7e/packages/react-native/React/CoreModules/RCTDeviceInfo.mm#L128-L129